var/spack/repos/builtin/packages/libvterm/package.py

```diff
@@ -3,6 +3,7 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
+
```

Comment:
drive by? Not sure if this is a problem from our style guides (I would assume not given that nothing failed)?

Reply:
what does "drive by" mean?

Reply:
non-intended change. I'm not sure if we have style guides that say "there should be exactly one empty line between the license identifier and the first statement of the file.

Reply:
Sure - happy to remove that line ... I did not notice.